ddtrace/utils/formats.py

```diff
@@ -89,3 +90,27 @@ def flatten_dict(d, sep=".", prefix=""):
         if isinstance(d, dict)
         else {prefix: d}
     )
+
+
+def parse_env_tags(envstr):
+    """Parse a string of tags typically provided via environment variables.
+
+    The expected string is of the form::
+
+        "key1:value1,key2:value2"
+    """
+    tags = {}
+    if not envstr:
+        return tags
+
+    for tag in envstr.split(","):
+        try:
+            key, value = tag.split(":", 1)
+        except ValueError as e:
+            e.args = (e.args or tuple()) + (tag,)
```

Comment:
I don't like this.. but I guess the alternative is subclassing `ValueError` and adding an argument/attribute for `tag` 😕.

Reply:
You can just raise a new `ValueError`, no?